build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -274,7 +275,7 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
                     .Environment("__WorkaroundCliCoreHostVer", hostVersion.LatestHostVersion.WithoutSuffix)
                     .Environment("__WorkaroundCliCoreHostBuildMajor", hostVersion.LatestHostBuildMajor)
                     .Environment("__WorkaroundCliCoreHostBuildMinor", hostVersion.LatestHostBuildMinor)
-                    .Environment("__WorkaroundCliCoreHostVersionTag", hostVersion.LatestHostPrerelease)
+                    .Environment("__WorkaroundCliCoreHostVersionTag", buildVersion.ReleaseSuffix)
```

Comment:
Why not use the `hostVersion.ReleaseSuffix` here and below?  That would make everything consistent.